src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer extends Truck implements MachineProducer {
+
+    @Override
+    public List<? super Truck> get() {
```

Comment:
it looks like we can add to result list some object of class Machine and Object (same for two other implemantations)
```suggestion
    public List<Truck> get() {
```